HealthMonitoring.AcceptanceTests/Helpers/Credentials.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+
+namespace HealthMonitoring.AcceptanceTests.Helpers
+{
+    class Credentials
+    {
+        public Guid MonitorId { get; set; }
```

Comment:
why MonitorId? The name does not reflect the purpose of it. We do not have multiple monitors in that sense and how about Admin creds that does have nothing with monitor process? Did you mean `Id`?